### PR TITLE
upload input files' info

### PR DESCRIPTION
**Pull request by yoouung**
- enhancement needed to transmit file object to server

### CODE REVIEW

**Comment by yoouung** [comment]
1. file upload
image

2. server response
image

**Comment by yoouung** [comment]
postman status
image

File: dt-demo-be/app.py [hunk 1]
```diff
@@ -95,6 +95,36 @@ def filter():
     else:
         return "Unexpected API request"
 
+@app.route("/api/filter/filelist", methods=["GET"])
```
Comment (doorooful) [inline dt-demo-be/app.py:98]: Is this line for file uploading?
I think It'd better to separate these changes to visualize file list

File: dt-demo-fe/src/components/file/uploadBtn.js [hunk 2]
```diff
@@ -1,38 +1,59 @@
...
-    const handleFileChange = (event) => {
-        setSelectedFile(event.target.files[0]);
-        onFileSelect(event.target.files[0]);
-    };
+    const handleFileUpload = async (e) => {
+        e.preventDefault();
+        setSelectedFile(e.target.files[0]);
+
+        if (selectedFile === null) {
+            console.log("No file selected");
```
Comment (doorooful) [inline dt-demo-fe/src/components/file/uploadBtn.js:26]: Could you use 'console.warn()' function in terms of handling unexpected behavior?
Or we may discuss further about the requirements!

**Review by doorooful — CHANGES_REQUESTED** [review]
I'd like to discuss about 2  things regarding comments!
1 - Separating this PR into 2 PRs
2 - Users' scenario and logging level
We may discuss tomorrow or further 🥹

File: dt-demo-be/app.py (hunk 1 above)
Comment (yoouung) [reply dt-demo-be/app.py]: uploaded separate PR for this issue

File: dt-demo-fe/src/components/file/uploadBtn.js (hunk 2 above)
Comment (yoouung) [reply dt-demo-fe/src/components/file/uploadBtn.js]: it is for when user clicked `cancel` button, so I just modified the log message.

**Review by doorooful — APPROVED** [review]
🥰

**Pull request merged**